js/login.js

```diff
@@ -0,0 +1,21 @@
+const loginEail = document.getElementById("email");
+const loginPw = document.getElementById("password");
+const loginBtn = document.getElementById("login-btn");
+
+function color() {
+    if(
+        loginEail.value.length > 0 && 
+        loginEail.value.indexOf("@") !== -1 && 
+        loginPw.value.length >=8
```

Comment:
이 조건을 읽기 쉽게 상수화해주는건 어떨까요? :)
혹은, regex로 패턴을 단순화해보는 시도도 할수있을것같아요! 

```suggestion
        const isEmailVaild = loginEail.value.length > 0 && loginEail.value.indexOf("@") !== -1 &&  loginPw.value.length >=8
```

[참고](https://piyush132000.medium.com/mastering-email-validation-in-javascript-multiple-approaches-ae718546160b)